packages/compile-loader/index.js

```diff
@@ -1,12 +1,21 @@
 const babelMerge = require('babel-merge');
 
-module.exports = (neutrino, options = {}) => {
+module.exports = (neutrino, { ruleId = 'compile', useId = 'babel', ...options } = {}) => {
+  if (neutrino.config.module.rules.has(ruleId)) {
+    throw new Error(
+      '@neutrinojs/compile-loader has been used twice with the same `ruleId`.\n' +
```

Comment:
Perhaps the other messages should be updated with this langauge (referencing "with the same `ruleId`")?

Reply:
I omitted the "same ruleId" part of the message for middleware that doesn't allow customisation of the ruleId, since it seemed misleading to hint that the middleware could be used with different ruleIds when that's not possible. However there may be a better way to word all of them - open to ideas :-)

Reply:
Oh gotcha – looks good then.